src/components/EventForm/formParts/RadioGroupButtons.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+import TextField from '@material-ui/core/TextField';
+import { createMuiTheme } from '@material-ui/core';
+import { ThemeProvider } from '@material-ui/styles'
+
+const radioButtonFontSize = '15px'
+const radioGroupLabelFontSize = '17px'
+
+const theme = createMuiTheme({
+  overrides: {
+      MuiFormControlLabel: {
+          label: {
+              fontSize: radioButtonFontSize,
+          }
+      }
+  }
+});
+
+
+export class RadioGroupButtons extends Component {
+
+  createButtons(optionsArr) {
+    
+    return optionsArr.map((option) => (option==='Other:') ?
```

Comment:
I thought we were going to use a boolean to control the inclusion of the other option?